scripts/dev.sh

```diff
@@ -105,7 +105,7 @@ sleep 8
 # OpenCog chatbot
 if [[ $OC_CHATBOT == 1 ]]; then
   tmux new-window -n 'relex_server' "cd $HR_WORKSPACE/opencog/relex/ && bash opencog-server.sh; $SHELL"
-  tmux new-window -n 'tel' "while true; do nc -zv localhost 17020 && break; sleep 1; done; expect $BASEDIR/load_scm.exp; $SHELL"
+  tmux new-window -n 'tel' "roslaunch opencog_control psi.launch & while true; do nc -zv localhost 17020 && break; sleep 1; done; expect $BASEDIR/load_scm.exp; $SHELL"
```

Comment:
Did you test it that this background process is going to be terminated when you stop it? 
Either create a new tmux window to run this launch file or add it to robot.launch with appropriate option. So it can be terminated correctly.